site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -34,19 +34,38 @@ An important configuration to note is the PodDisruptionBudget fields, `agones.ex
 
 ## Deployment Considerations
 
-When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as `agones-controller` and 2 `agones-extensions` pods. For example: 
+Leader election will automatically be enabled when `SplitControllerAndExtensions` is enabled and `agones.controller.replicas` is > 1. [`agones.controller.replicas`]({{< relref "/docs/Installation/Install Agones/helm.md#configuration" >}}) defaults to 2.
```

Comment:
I am confused. What I meant previously was that you had a `publishVersion` embedded in the line, but the whole think was next to a multi-line `publishVersion`. This change instead moved the words out of the `publishVersion` and so now you have the old problem - you need to wrap this.

What I was intending was something like:

```
{{% feature expiryVersion="1.31.0" %}}
The old stuff goes here
{{% /end %}}
{{% feature publishVersion="1.31.0" %}}
The new stuff goes here
{{% /end %}}
```

*However*, that said, at this point you might just remove it all and stick to the new stuff - if you wait until tomorrow this will all be moot as the 1.31 release will be shipped.

Reply:
I will go with the option of remove it all and stick it to the new stuff and wait till tomorrow to merge.